app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split(" ")
+    path_2 = command_parts[2].split("/")
```

Comment:
The code assumes that the command will always have at least three parts and that the third part will always be a valid path. Consider adding error handling to manage cases where the command does not meet these expectations.